### PR TITLE
✨ Add function which checks if all players revealed two cards 

### PR DESCRIPTION
**Pull request by fre-ben**
- This function is needed for the pre-game start
- Checks with each clicked card by any player if all players have 2 cards revealed

Next up is a function which will check the highest score of the two revealed cards.
Player with the highest score of two revealed cards will take the first turn in the upcoming round.

Also I renamed yesterday's function `checkIfTwoCardsAreRevealed` to `checkTwoCardsRevealed`.
I really would like to name them shorter, but I'm afraid to go full [ninja](https://javascript.info/ninja-code) 🌚 

Deployment: https://sisu-pipelin-allplayerc-hatt42.herokuapp.com/
Fixes parts of #73

### CODE REVIEW

**Review by mewiemers — APPROVED** [review]
LGTM➕ like the new readme.md

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # SISU &middot; [![Build Status](https://img.shields.io/travis/npm/npm/latest.svg?style=flat-square)](https://travis-ci.org/npm/npm) [![npm](https://img.shields.io/npm/v/npm.svg?style=flat-square)](https://www.npmjs.com/package/npm) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/your/your-project/blob/master/LICENSE)
+
```
Comment (JoscaWij) [inline README.md:2]: The readme shouldn't be part of this pull request. Maybe cherry pick the commit for the readme :)

**Review by JoscaWij — APPROVED** [review]
Lgtm 👍 great job - I am not a pro on socket.io though 

Just one minor thing 😉
